hacl-rs/src/bignum.rs

```diff
@@ -0,0 +1,12 @@
+pub mod base;
+pub mod bignum256;
+pub mod bignum256_32;
+pub mod bignum32;
+pub mod bignum4096;
+pub mod bignum4096_32;
+pub mod bignum64;
+pub mod bignum_base;
+
+pub mod test {
```

Comment:
This kind of code we may want to delete when we know that we don't update a lot anymore.